pom.xml

```diff
@@ -2113,7 +2115,7 @@
               <include>**/*Suite.java</include>
             </includes>
             <reportsDirectory>${project.build.directory}/surefire-reports</reportsDirectory>
-            <argLine>-ea -Xmx3g -Xss4m -XX:ReservedCodeCacheSize=${CodeCacheSize}</argLine>
+            <argLine>-ea -Xmx6g -Xss4m -XX:ReservedCodeCacheSize=${CodeCacheSize}</argLine>
```

Comment:
Required now to make TimSort tests pass in Java 11, probably because of how direct buffer handling has changed.

Reply:
It would be good to know what changed. Do you have an idea?

Reply:
What tests failed?

Reply:
The TimSort test failed, where it allocates a very very large array.
I'm pretty sure it's because we can't get around the off-heap allocation limits in Java 9+ (see https://github.com/apache/spark/pull/22993/files). I think this would also pass if we manually set the off-heap limit very high, or allowed access to the `Cleaner` class on the command line. Upping the memory seemed like a simpler workaround but those should be fine too.

Reply:
the % changed is a quite a bit though

Reply:
+1 for increasing this. I also saw multiple TimSort OOM failures even in Apache Spark Jenkins environment(JDK8).

Reply:
@felixcheung Let me try 4G or 5G. I think I just doubled it and it worked so I left it, but may not be necessary to make it that big.